### ISSUE TITLE
Proportions covered is empty

### ISSUE THREAD

**Issue by JoannaTan** [brwnj/covviz#6]
Hi,

I am using covviz to visualize the coverage across multiple samples.

However, the html file that I obtained does not have anything for the Proportions Covered tab.

This is the command that I used to run covviz

```
covviz indexcov.bed --ped samplechr.ped --sample-col sample_id --sex-col sex --output 100samples.html
```

I am running covviz using singularity.

Am I supposed to pass in the .roc file? 

Thank you very much.

Hope to hear from you soon. 

Best regards,
Joanna

**Comment by brwnj** [comment]
The latest version (1.1.1) no longer requires the .roc file. Can you verify this is a problem with the latest version?

**Comment by JoannaTan** [comment]
Hi @brwnj,

I have checked the version of covviz that I am using and it is version 1.1.1

```
covviz                    1.1.1                    pypi_0    pypi
```

**Comment by brwnj** [comment]
I added some indexcov output (2c7e4fd6a4aaa7aa6bb46573bc0cad6cdc17357f) that I have been using in testing the CLI. Can you verify the format of your indexcov.bed output against what's in covviz/data?

**Comment by JoannaTan** [comment]
Hi @brwnj,

Thank you for your help.

I realized that my indexcov.bed has the chr appended to the chromosome. 

```
chr1    0       16384
```

I removed the chr and now it works.

Thank you very much.

**Comment by brwnj** [comment]
Thanks for reporting this because it's definitely a bug. I'll take a look
at it tomorrow.

**Comment by brwnj** [comment]
fixed in 6cee85db1f831335e0d4d214c6ec725f1d229677.
